packages/components/organisms/f-loyalty/src/components/_tests/HowItWorks.test.js

```diff
@@ -0,0 +1,157 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import VueI18n from 'vue-i18n';
+import i18nMocker from './helper';
+import HowItWorksLayout from '../HowItWorks.vue';
+
+import enGB from '../../tenants/en-GB';
+import enAU from '../../tenants/en-AU';
+
+const localVue = createLocalVue();
+localVue.use(VueI18n);
+
+let messages,
+    wrapper;
+
+beforeEach(() => {
+    jest.resetAllMocks();
+
+    messages = {
+        'en-GB': enGB,
+        'en-AU': enAU
+    };
+});
+
+describe('HowItWorksLayout', () => {
+    it('should be defined', () => {
+        // Arrange
+        const i18n = {
+            locale: 'en-AU',
+            fallbackLocale: 'en-AU',
+            messages
+        };
+
+        // Act
+        wrapper = shallowMount(HowItWorksLayout, {
+            localVue,
+            i18n,
+            mocks: {
+                $t: key => i18nMocker(key, i18n.locale)
+            }
+        });
+
+        // Assert
+        expect(wrapper).toBeDefined();
+    });
+
+    describe('when in AU', () => {
+        beforeEach(() => {
+            // Arrange
+            const i18n = {
+                locale: 'en-AU',
+                fallbackLocale: 'en-AU',
+                messages
+            };
+
+            // Act
+            wrapper = shallowMount(HowItWorksLayout, {
+                localVue,
+                i18n,
+                mocks: {
+                    $t: key => i18nMocker(key, i18n.locale)
+                }
+            });
+        });
+
+        it('bags should be calculated as expected', () => {
+            // Assert
+            expect(wrapper.vm.bags).toEqual([
```

Comment:
Can do this in another PR, but this looks like a good candidate for a snapshot test? Same for the below.

Reply:
good point will note it down thanks